ProcessMaker/Http/Controllers/Api/TemplateController.php

```diff
@@ -254,7 +266,6 @@ private function checkIfAssetsExist($request)
 
     protected function createProcess(Request $request)
     {
-        $request->validate(Process::rules($request->id));
```

Comment:
@estebangallego , this was the original line:

    $request->validate(Template::rules($request->id, $this->types['process'][4]));

This line was responsible for applying the rules defined in `Template::rules` to the `process_templates` table:

    'name' => ['required', $unique, 'alpha_spaces', 'max:255'],
    'description' => 'required',
    'version' => ['required', 'regex:/^[0-9.]+$/'],

I think the most appropriate action would be to reintegrate that line or define specific rules such as:

    $request->validate(ProcessTemplate::rules($request->id));

Reply:
I also think we have a similar issue with `Screens`:

    $request->validate(Screen::rules($request->id));

It should be `ScreenTemplate::rules`...

[You can view the related commit here](https://github.com/ProcessMaker/processmaker/commit/52a0bd200859939e98fb67d98b5fb8e0f6da21e9)

Reply:
Thank you @eiresendez! Yes, we can review it in more detail tomorrow. However, in this context, we are using the createProcess() function to generate processes in multiple scenarios, using different $request inputs. But you are right, I added the previous validation, and it works correctly.


`$request->validate(Template::rules($request->id, $this->types['process'][4]));`